docs/features/bonding-quick-start.md

```diff
@@ -0,0 +1,139 @@
+
+This is accomplished using the [socket groups](./socket-groups.md) introduced in [SRT v1.5](https://github.com/Haivision/srt/releases/tag/v1.5.0). The general concept of socket groups means having a group that contains multiple sockets, where one operation for sending one data signal is applied to the group. Single sockets inside the group will take over this operation and do what is necessary to deliver the signal to the receiver.
+
+Two modes are supported:
+
+- [Broadcast](./socket-groups.md#1-broadcast) - In *Broadcast* mode, data is sent redundantly over all the member links in a group. If one of the links fails or experiences network jitter and/or packet loss, the missing data will be received over another link in the group. Redundant packets are simply discarded at the receiver side.
+
+- [Main/Backup](./bonding-main-backup.md) - In *Main/Backup* mode, only one (main) link at a time is used for data transmission while other (backup) connections are on standby to ensure the transmission will continue if the main link fails. The goal of Main/Backup mode is to identify a potential link break before it happens, thus providing a time window within which to seamlessly switch to one of the backup links.
+
+While this feature remains disabled by default, you can use the [`ENABLE_BONDING=ON`](../build/build-options.md#enable_bonding) build option to activate it.
+
+## 2. Additional Documentation and Code Examples
+
+- [SRT Connection Bonding: Introduction](./bonding-intro.md) - Introduction to Connection Bonding. Description of group (bonded) connections.
+- [SRT Connection Bonding: Socket Groups](./socket-groups.md) - Description of socket groups in SRT. Here you will also find information regarding the `srt-test-live` application for testing Connection Bonding.
+- [SRT Connection Bonding: Main/Backup](./bonding-main-backup.md) - Main/Backup mode description.
+- Refer also to the [SRT API](../API/API.md) and [SRT API Functions](../API/API-functions.md#socket-group-management) documentation for updates related to bonding.
+- Code examples: simple [client](../../examples/test-c-client-bonding.c) and [server](../../examples/test-c-server-bonding.c) implementation.
+
+## 3. Quick Start Guidelines
+
+### 3.1. Transition to a Bonded Connection
+
+For the end-user of the SRT library, operations on a group of bonded sockets should be very much like on a single SRT socket. It is important that the application needs to check regularly for broken sockets in a group and to reconnect broken links. Refer to the [SRT API Functions](../API/API-functions.md#socket-group-management) document to learn what functions can be used with a socket group.
+
+**On the listener side (pseudocode):**
+
+* Create a socket: `sock_id = srt_create_socket()`,
+* `srt_bind(sock_id, ..)` to an interface (as usual for a listener socket),
+* `srt_setsockflag(sock_id, SRTO_GROUPCONNECT, &(int yes = 1), sizeof yes)` (by allowing group connections your app confirms being group aware),
```

Comment:
To avoid someone taking this pseudocode seriously.
```suggestion
* `srt_setsockflag(sock_id, SRTO_GROUPCONNECT, &yes /*int yes = 1*/, sizeof yes)` (by allowing group connections your app confirms being group aware),
```